.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,33 @@
+---
+name: ✨ Feature request
+about: Suggest an idea for the Molecule project
+---
+
+# Issue Type
+
+- Feature request
+
+# Molecule and Ansible details
+
+```
+ansible --version
```

Comment:
make it: 
```
--version && molecule --version
```
It takes too much space to separate them for no real benefits.

Reply:
Excellent idea, thank you